<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UIMain</class>
 <widget class="QMainWindow" name="UIMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QWBFS Manager</string>
  </property>
  <property name="iconSize">
   <size>
    <width>24</width>
    <height>24</height>
   </size>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QSplitter" name="sViews">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="PartitionWidget" name="pwMainView" native="true"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="windowTitle">
    <string>Tools</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTabWidget" name="twPages">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="movable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="wFS">
        <attribute name="icon">
         <iconset resource="../resources/resources.qrc">
          <normaloff>:/icons/view-list-tree.png</normaloff>:/icons/view-list-tree.png</iconset>
        </attribute>
        <attribute name="title">
         <string>File System</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QComboBox" name="cbDrives"/>
         </item>
         <item row="0" column="1">
          <widget class="QToolButton" name="tbReloadDrives">
           <property name="icon">
            <iconset resource="../resources/resources.qrc">
             <normaloff>:/icons/view-refresh.png</normaloff>:/icons/view-refresh.png</iconset>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QSplitter" name="splitter">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QTreeView" name="tvFolders">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="uniformRowHeights">
             <bool>true</bool>
            </property>
            <property name="headerHidden">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QListView" name="lvFiles">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="dragEnabled">
             <bool>true</bool>
            </property>
            <property name="dragDropMode">
             <enum>QAbstractItemView::DragOnly</enum>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::ExtendedSelection</enum>
            </property>
            <property name="uniformItemSizes">
             <bool>true</bool>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="wExport">
        <attribute name="icon">
         <iconset resource="../resources/resources.qrc">
          <normaloff>:/icons/document-export.png</normaloff>:/icons/document-export.png</iconset>
        </attribute>
        <attribute name="title">
         <string>Export</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QToolButton" name="tbClearExport">
             <property name="toolTip">
              <string>Clear the list</string>
             </property>
             <property name="icon">
              <iconset resource="../resources/resources.qrc">
               <normaloff>:/icons/edit-clear-list.png</normaloff>:/icons/edit-clear-list.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="tbRemoveExport">
             <property name="toolTip">
              <string>Remove the selected items from the list</string>
             </property>
             <property name="icon">
              <iconset resource="../resources/resources.qrc">
               <normaloff>:/icons/list-remove.png</normaloff>:/icons/list-remove.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="tbExport">
             <property name="toolTip">
              <string>Process the export list</string>
             </property>
             <property name="icon">
              <iconset resource="../resources/resources.qrc">
               <normaloff>:/icons/document-export.png</normaloff>:/icons/document-export.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QListView" name="lvExport">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="dragDropMode">
            <enum>QAbstractItemView::DropOnly</enum>
           </property>
           <property name="defaultDropAction">
            <enum>Qt::CopyAction</enum>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::ExtendedSelection</enum>
           </property>
           <property name="verticalScrollMode">
            <enum>QAbstractItemView::ScrollPerPixel</enum>
           </property>
           <property name="horizontalScrollMode">
            <enum>QAbstractItemView::ScrollPerPixel</enum>
           </property>
           <property name="uniformItemSizes">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="aReloadPartitions"/>
   <addaction name="aAbout"/>
   <addaction name="aQuit"/>
  </widget>
  <action name="aReloadPartitions">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/reload-48.png</normaloff>:/icons/reload-48.png</iconset>
   </property>
   <property name="text">
    <string>Reload partitions</string>
   </property>
  </action>
  <action name="aAbout">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/about-48.png</normaloff>:/icons/about-48.png</iconset>
   </property>
   <property name="text">
    <string>About...</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="aQuit">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/exit-48.png</normaloff>:/icons/exit-48.png</iconset>
   </property>
   <property name="text">
    <string>Quit...</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PartitionWidget</class>
   <extends>QWidget</extends>
   <header>PartitionWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
